closed/test/jdk/openj9/internal/security/property-java.security

```diff
@@ -36,7 +36,7 @@ RestrictedSecurity.TestBase.Version.jce.certpath.disabledAlgorithms =
 RestrictedSecurity.TestBase.Version.jce.legacyAlgorithms =
 RestrictedSecurity.TestBase.Version.jce.provider.1 = sun.security.provider.Sun
 RestrictedSecurity.TestBase.Version.jce.provider.2 = com.sun.crypto.provider.SunJCE
-RestrictedSecurity.TestBase.Version.jce.provider.3 = sun.security.ssl.SunJSSE
+RestrictedSecurity.TestBase.Version.jce.provider.3 = com.sun.net.ssl.internal.ssl.Provider
```

Comment:
Isn't a change to `RestrictedSecurity.TestBase.Version.desc.hash` required because of this update?

Reply:
In the `TestProperties.java` file, all profiles except for `Test-Profile-SameStartWithoutVersion` are designed to test error messages. In these profiles, the error message is returned before the hash value is checked, so the hash value is not used.

Reply:
It would be helpful to have comments to that effect directly in the property files and perhaps specify hashes of all zeros if they are not meaningful.

Reply:
Or they could say something like this:
```
RestrictedSecurity.Test-Profile.Base.desc.hash = SHA256:not-checked
```

Reply:
Since we don’t use these hash values, I think it’s fine to just leave them as they are. However, if we do want to make them more readable, like changing them to "`SHA256:not-checked`", I can update this by creating another PR. For two reasons:

- This change "`SHA256:not-checked`" would need to be ported to all JDK versions.

- I can combine this update with the "fixing/improving these comments" which suggested on the JDK8 PR https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/784, starting from the head stream.

Reply:
Issue created: https://github.ibm.com/runtimes/jit-crypto/issues/615

Reply:
Please create an issue in a public repository, perhaps at https://github.com/eclipse-openj9/openj9/issues.